Corona-Warn-App/src/main/res/layout/include_submission_dispatcher.xml

```diff
@@ -54,6 +75,18 @@
                 app:layout_constraintStart_toStartOf="@+id/guideline_card_start"
                 app:layout_constraintTop_toBottomOf="@+id/submission_dispatcher_qr" />
 
+            <TextView
+                android:id="@+id/submission_dispatcher_already_positive_text"
```

Comment:
The distance between the first title, and the first card, and the second title and the third card is inconsistent.

Reply:
![Screenshot from 2020-11-19 19-40-48](https://user-images.githubusercontent.com/1439229/99709356-5f2ec780-2a9f-11eb-9764-e8a9f43b10b0.png)

Reply:
@d4rken for some reason I am only seeing this now. I'll resolve it immediately. 👍